### ISSUE TITLE
Missing algorithms supported for Forecasting Class

### ISSUE THREAD

**Issue by jpmlblog** [Azure/MachineLearningNotebooks#1267]
[Enter feedback here]
The following site says that the Forecasting class supports **"Elastic Net, Light GBM, Gradient Boosting, Decision Tree, K Nearest Neighbors, LARS Lasso, Stochastic Gradient Descent (SGD), Random Forest, Extremely Randomized Trees, Xgboost, Auto-ARIMA, Prophet, ForecastTCN"** algorisms.

https://docs.microsoft.com/en-us/python/api/azureml-train-automl-client/azureml.train.automl.constants.supportedmodels?view=azure-ml-py

However, only **"AutoArima, Average, Naive, Prophet, SeasonalAverage, SeasonalNaive, TCNForecaster"** algorithms are written on the following site. What is this difference?

https://docs.microsoft.com/en-us/python/api/azureml-train-automl-client/azureml.train.automl.constants.supportedmodels.forecasting?view=azure-ml-py

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b06f6d30-ecfb-1a44-951d-e1a6214ed6e1
* Version Independent ID: 5ef8c02b-8d44-a646-d6f8-b3d9c3251e0e
* Content: [azureml.train.automl.constants.SupportedModels.Forecasting class - Azure Machine Learning Python](https://docs.microsoft.com/en-us/python/api/azureml-train-automl-client/azureml.train.automl.constants.supportedmodels.forecasting?view=azure-ml-py)
* Content Source: [AzureML-Docset/stable/docs-ref-autogen/azureml-train-automl-client/azureml.train.automl.constants.SupportedModels.Forecasting.yml](https://github.com/MicrosoftDocs/MachineLearning-Python-pr/blob/live/AzureML-Docset/stable/docs-ref-autogen/azureml-train-automl-client/azureml.train.automl.constants.SupportedModels.Forecasting.yml)
* Service: **machine-learning**
* Sub-service: **core**
* GitHub Login: @DebFro
* Microsoft Alias: **debfro**

**Comment by Aniththa** [comment]
Hi there! Thanks for reaching out. Here is a complete list of supported algos for forecasting:https://docs.microsoft.com/en-us/azure/machine-learning/how-to-configure-auto-train#supported-models

This link outlines algos only available to forecasting. (Please note, regression algos can also be used for forecasting): https://docs.microsoft.com/en-us/python/api/azureml-train-automl-client/azureml.train.automl.constants.supportedmodels.forecasting?view=azure-ml-py

Apologies for any confusion! We are in the process of cleaning that documentation. 
Please feel free to reach out if you have any other questions.

**Comment by jpmlblog** [comment]
Hi @Aniththa , thank you for the comment. Let me ask you one more question.
**"Average, Naive, SeasonalAverage, SeasonalNaive"** algorithms are not written in the Forecasting of the following site, why is this?
https://docs.microsoft.com/en-us/azure/machine-learning/how-to-configure-auto-train#supported-models

**Comment by CESARDELATORRE** [comment]
Fyi, @Aniththa , we might want to update the page above so it also mentions those algorithms.

**Comment by CESARDELATORRE** [comment]
I confirmed this is being tracked by our docs team and will be fixed around 2/9.

**Comment by cartacioS** [comment]
Hello,

While the algorithms listed have infact been introduced to the SDk, we actually still performing internal validation on the models before we document them on ms docs, samples, and introduce them to the UI. While "technically" you can enable these models today, you may hit some issues while we complete validation. Once this is done, we will promptly update the corresponding areas to reflect the other models.

Thanks!

**Comment by nibaccam** [comment]
@jpmlblog we are tracking this update internally and will now proceed to close this thread. Thank you for reporting this issue. 
#please-close
